docs/dev/fundamentals/zkSync.md

```diff
@@ -0,0 +1,90 @@
+- The proof for the block is submitted to the zkSync smart contract as block verification. If the verification succeeds, the new state is considered finalized.
+
+Furthermore, on zkSync, each L2 block will progress through the following four stages until it is finalized.
+
+- `Pending`: The transaction was received by the operator, but it has not been processed yet.
+- `Processed`: The transaction is processed by the operator and is confirmed to be included in the next block.
+committed indicates that the transaction data of this block has been posted on Ethereum. It does not prove that it has been executed in a valid way, but it ensures the availability of the block data.
+- `Finalized`: This indicates that the SNARK validity proof for the transaction has been submitted and verified by the smart contract. After this step, the transaction is considered to be final.
+
+The typical time for a transaction to go from `Processed` to `Finalized` is a couple of hours at the current stage.
+
+Please note that for developer convenience, we usually treat the `Processed` and `Committed` states as a single stage called `Committed` since they have no difference from the UX/DexEx standpoints.
+
+### The State of the Rollup
+The current version of zkSync 2.0 solves the needs of most applications on Ethereum, and with more features planned for release soon, zkSync 2.0 will provide developers with a design space to experiment with applications not possible on Ethereum today. With this release, we are supporting the following features:
+
+- Native support of ECDSA signatures: Unlike the first version of zkSync and most of the ZK Rollups, no special operation is required to register the user’s private key. Any account can be managed in L2 with the same private key that is used for L1.
+- Solidity 0.8.x support: Deploy your existing codebase with little to no changes required.
+Web3 API. With small exceptions, our API is fully compatible with Ethereum. This allows seamless integration with existing indexers, explorers, etc.
+- Support for Ethereum cryptographic primitives: zkSync natively supports `keccak256`, `sha256`, and `ecrecover` via precompilers.
+- Hardhat plugin: Allows easy testing and development of smart contracts on zkSync.
+- L1 → L2 smart contract messaging: Allows developers to pass data from Ethereum L1 to smart contracts on zkSync, providing the required information to run various smart contracts.
+Some features are not included in our current testnet that we’re looking to ship in future upgrades. In estimated chronological order, this includes:
+
+
+- Account abstraction: Imagine being able to implement custom logic for signature checking for your account. Or maybe social recovery? Currently, on most of the EVM chains, users need to deploy smart contract wallets for such purposes. All of this would be easily supported with account abstraction.
```

Comment:
Account abstraction is already included. Even though some breaking changes are possible, it is included already